src/index.js

```diff
@@ -21,15 +32,50 @@ import "./index.scss";
 //> Components
 // Root component
 import App from "./App";
+//> Root Reducer
+import rootReducer from "./store/reducers";
 //> Service Worker
 import registerServiceWorker from "./registerServiceWorker";
 //#endregion
 
+//#region > Redux Store Initialization
+const INTEL = new Intel();
+
+//#TODO
+// Must be moved to INTEL in future?
+const getIntel = () => {
+  return INTEL;
+};
+
+const composeEnhancers =
+  typeof window === "object" &&
+  process.env.NODE_ENV !== "production" &&
+  window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+    ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
+        // Specify extension’s options like name, actionsBlacklist, actionsCreators, serialize...
+      })
+    : compose;
+
+const enhancer = composeEnhancers(
+  applyMiddleware(
+    loadingBarMiddleware(),
+    thunk.withExtraArgument({
+      getIntel, // Intel
```

Comment:
Is this a valid comment?
```suggestion
      // Intel
      getIntel,
```

Reply:
It makes sense to comment the those params the way proposed by @schettn , yes.